### PR TITLE
add a "one pager" to the CI Signal handbook

### PR DESCRIPTION
**Pull request by leonardpahlke**
#### What type of PR is this:
/kind documentation

#### What this PR does / why we need it:

Currently, it is difficult to get a clear picture of what CI Signal is doing without reading 20 pages of the CI Signal manual. By adding a "one-pager" manual that provides an overview of responsibilities, the onboarding experience and team responsibilities can be more clearly communicated (see issue https://github.com/kubernetes/sig-release/issues/1857). This topic has been discussed in a sig-release meeting on 2022-03-22 see [agenda](https://docs.google.com/document/d/1Fu6HxXQu8wl6TwloGUEOXVzZ1rwZ72IAhglnaAMCPqA/edit)

#### Which issue(s) this PR fixes:

Fixes: https://github.com/kubernetes/sig-release/issues/1857

#### Special notes for your reviewer:

We are currently working on some additional videos which will be added later (see issue https://github.com/kubernetes/sig-release/issues/1856).

cc @kubernetes/ci-signal

### CODE REVIEW

**Comment by leonardpahlke** [comment]
cc @RobertKielty

**Comment by mkorbi** [comment]
Why not just add/update https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/ci-signal#overview-of-ci-signal-responsibilities ? Because I don't read it as an onboarding document, it is an overview of tasks. Also, it would need to be made clear the difference between this section of [onboarding](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/ci-signal#onboarding).

Sprawling information, making it redundant, and maybe being formalized in different explanations is more counterproductive.

**Comment by leonardpahlke** [comment]
The idea was to have a one pager that we can share to ones interested in the team and to everybody who is interested applying for ci signal shadow.
It can also be used as onboarding sheet but I agree there is aleady a section in the current handbook. So maybe rename it? 

release-team/role-handbooks/ci-signal/onboarding.md -> release-team/role-handbooks/ci-signal/one-pager.md

and write **ONE PAGER** ro something in the header

cc @kubernetes/ci-signal

File: release-team/role-handbooks/ci-signal/one-pager.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# CI Signal Responsibility Summary
+
+The scope of CI Signal team is to monitor the results from our CI and raise any issues we find. 
+
+Detailed information can be found in  the [CI Signal handbook](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/ci-signal/README.md), but essentially the job consists of the following.
+
+1. Monitor Testgrid: The team is particularly interested the [testgrid dashboards](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/ci-signal/README.md#blocking-vs-informing-dashboards): [master-blocking](https://testgrid.k8s.io/sig-release-master-blocking) & [master-informing](https://testgrid.k8s.io/sig-release-master-informing).
```
Comment (mkorbi) [inline release-team/role-handbooks/ci-signal/one-pager.md:7]: The team is particularly interested in the [testgrid dashboards]

File: release-team/role-handbooks/ci-signal/one-pager.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+2. Create issues on GitHub: [Create GitHub issues](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/ci-signal/README.md#opening-issues) reporting [*failing*](https://github.com/kubernetes/kubernetes/issues/new?assignees=&labels=kind%2Ffailing-test&template=failing-test.yaml) or [*flaky*](https://github.com/kubernetes/kubernetes/issues/new?assignees=&labels=kind%2Fflake&template=flaking-test.yaml) tests.
+3. Track GitHub issues: Follow up on github issues, talk to [other sigs](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/ci-signal/README.md#working-with-sigs-outside-sig-release), ping and cc people.You may add the [milestone](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/ci-signal/README.md#working-with-sigs-outside-sig-release) or a [priority lables](https://github.com/kubernetes/sig-release/blob/master/release-team/role-handbooks/ci-signal/README.md#priority-labels) to the github issue.
```
Comment (mkorbi) [inline release-team/role-handbooks/ci-signal/one-pager.md:9]: needs a space between
"ping and cc people. You may add"

**Review by mkorbi — CHANGES_REQUESTED** [review]

**Review by mkorbi — APPROVED** [review]
lgtm

**Pull request merged**